core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.spec.ts

```diff
@@ -1,235 +1,227 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
+
 import { CommonModule } from '@angular/common';
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { AbstractControl, ReactiveFormsModule } from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
 
 import { ButtonModule } from 'primeng/button';
 
+import { DotMessageService } from '@dotcms/data-access';
 import { DotCMSContentTypeLayoutRow } from '@dotcms/dotcms-models';
 import { DotMessagePipe } from '@dotcms/ui';
+import { MockDotMessageService } from '@dotcms/utils-testing';
 
 import { DotEditContentFormComponent } from './dot-edit-content-form.component';
 
 import { DotEditContentFieldComponent } from '../dot-edit-content-field/dot-edit-content-field.component';
 import { FIELD_MOCK } from '../dot-edit-content-field/dot-edit-content-field.component.spec';
 
-describe('DotFormComponent', () => {
-    let component: DotEditContentFormComponent;
-    let fixture: ComponentFixture<DotEditContentFormComponent>;
-
-    beforeEach(async () => {
-        await TestBed.configureTestingModule({
-            imports: [
-                CommonModule,
-                ReactiveFormsModule,
-                DotEditContentFieldComponent,
-                ButtonModule,
-                DotMessagePipe
-            ]
-        }).compileComponents();
-
-        fixture = TestBed.createComponent(DotEditContentFormComponent);
-        component = fixture.componentInstance;
-        fixture.detectChanges();
-    });
-
-    it('should create', () => {
-        expect(component).toBeTruthy();
-    });
-});
-describe('DotEditContentFormComponent', () => {
-    let component: DotEditContentFormComponent;
-    let fixture: ComponentFixture<DotEditContentFormComponent>;
-
-    beforeEach(async () => {
-        await TestBed.configureTestingModule({
-            imports: [
-                CommonModule,
-                ReactiveFormsModule,
-                DotEditContentFieldComponent,
-                ButtonModule,
-                DotMessagePipe
-            ],
-            declarations: [DotEditContentFormComponent, DotEditContentFieldComponent]
-        }).compileComponents();
-
-        fixture = TestBed.createComponent(DotEditContentFormComponent);
-        component = fixture.componentInstance;
-        fixture.detectChanges();
-    });
-
-    it('should create', () => {
-        expect(component).toBeTruthy();
-    });
-
-    describe('initilizeForm', () => {
-        it('should initialize the form group with form controls for each field in the `formData` array', () => {
-            const formData: DotCMSContentTypeLayoutRow[] = [
-                {
-                    divider: {
-                        clazz: 'com.dotcms.contenttype.model.field.ImmutableRowField',
+export const LAYOUT_MOCK: DotCMSContentTypeLayoutRow[] = [
```

Comment:
lets create a utils folder and a mocks.ts file and drop all the mocks there if we can reuse it.